packages/frontend/src/components/navbar/SidebarMenu.tsx

```diff
@@ -11,7 +11,7 @@ export function SidebarMenu() {
       <div
         id="sidebar-menu"
         className={classNames(
-          'hidden fixed top-0 left-0 z-50',
+          'hidden fixed top-[32px] left-0 z-50',
```

Comment:
Would be good to have this conditional on config. Also 32px is part of tailwind's scale so you could use top-8